parameter_files/fates_params_default.cdl

```diff
@@ -1445,7 +1445,7 @@ data:
 
  fates_maintresp_model = 1 ;
 
- fates_mort_disturb_frac = 1 ;
```

Comment:
@JunyanDing can we revert this change? If we change the default file for a non-hydro parameter, I would like to see that change in a separate pull request.

Reply:
Chatting with @JunyanDing, we both agree to revert this change too.

Reply:
Reverted